foodalert/sender.py

```diff
@@ -31,7 +31,7 @@ def send_email(body, recipients, time, location):
 
         send_db_mail(
             slug=time,
-            recipient=settings.SAFE_EMAIL_RECIPIENT,
+            recipient=settings.FA_DISPLAYED_EMAIL_RECIPIENT,
```

Comment:
delete this to see if undisclosed-recipients shown on the "To:" field of email sent